packages/@aws-cdk/aws-apigatewayv2-integrations/test/http/integ.aws-integration-startSyncExecution.ts

```diff
@@ -0,0 +1,29 @@
+import { HttpApi } from '@aws-cdk/aws-apigatewayv2';
+import { StateMachine, Chain, StateMachineType, Pass } from '@aws-cdk/aws-stepfunctions';
+import { App, CfnOutput, Stack } from '@aws-cdk/core';
+import { StepFunctionsStartSyncExecutionIntegration } from '../../lib';
+
+/*
+ * Stack verification steps:
+ * "curl <endpoint-in-the-stack-output>" should return 'success'
+ */
+
+const app = new App();
+
+const stack = new Stack(app, 'integ-aws-service');
+
+const state = new StateMachine(stack, 'MyStateMachine', {
+  definition: Chain.start(new Pass(stack, 'Pass')),
+  stateMachineType: StateMachineType.EXPRESS,
+});
+
+const endpoint = new HttpApi(stack, 'AwsIntegrationApi', {
+  defaultIntegration: new StepFunctionsStartSyncExecutionIntegration(stack, {
+    stateMachine: state,
+    input: '$request.body',
+  }),
+});
```

Comment:
same question as above.

Reply:
It is integration testing. While testing, it validates the test against expected json.